sdk/spring/azure-spring-boot-test-aad/src/test/java/com/azure/test/aad/approle/AADAppRoleStatelessAuthenticationFilterIT.java

```diff
@@ -88,13 +88,14 @@ public void testAADAppRoleStatelessAuthenticationFilter() {
     @EnableGlobalMethodSecurity(prePostEnabled = true)
     @SpringBootApplication
     @RestController
-    public static class DumbApp extends WebSecurityConfigurerAdapter {
+    public static class DumbApp extends AzureOAuth2Configuration {
```

Comment:
For `AADAppRoleStatelessAuthenticationFilter`, we do not need to use `AzureOAuth2Configuration`.